_layouts/candidate.html

```diff
@@ -19,7 +19,19 @@ <h2>{{ candidate.name }}</h2>
         <div class="candidate__info-container">
           <img class="candidate__photo" src="{{ candidate.photo_url | default: 'https://s3-us-west-1.amazonaws.com/odca-candidate-photos/no-image.png' }}" alt="{{ candidate.name }}" />
           <div class="candidate__info">
-            <div class="candidate__occupation">{{ candidate.occupation }}</div>
+            <div class="candidate__occupation__commitee_name">{% if candidate.is_incumbent == true and candidate.committee_name != null %}
```

Comment:
The class name should be `candidate__occupation-committee-name` to follow the convention `<block-name>__<element-name>`